.github/workflows/ci.yaml

```diff
@@ -26,15 +26,17 @@ jobs:
         run: |
           mvn -V -B -s .github/mvn-settings.xml clean verify -DexcludeTags=product,native,codequarkus
       - name: Zip Artifacts
+        if: failure()
         run: |
-          zip -r artifacts-jvm${{ matrix.java }}.zip . -i '*-reports/*' '*/archived-logs/*'
+          zip -r linux-build-released-jvm${{ matrix.java }}.zip . -i '*-reports/*' '*/archived-logs/*'
       - name: Archive artifacts
         uses: actions/upload-artifact@v1
+        if: failure()
         with:
           name: ci-artifacts
```

Comment:
this should be adjusted to reflect to which build the artifact belongs

atm there is 4x `name: ci-artifacts`

Reply:
This configuration results in a single ci-artifact with up to four zip files (one per check). 
I can configure to have four artifacts, each artifact with one zip file, but I think it's better to download one file and extract all the check outputs at once ...

Reply:
aha, ok, I thought it would get overridden